kvmagent/kvmagent/plugins/zbs_storage_plugin.py

```
@@ -4,23 +4,259 @@
+
+    def _is_luks_volume(self, install_path, encrypted_dek=None):
+        try:
+            if encrypted_dek:
+                with volume_secret.luks_secret_channel(encrypted_dek) as secret_file:
+                    cmd = '/usr/bin/qemu-img info --object secret,id=luks_sec,format=raw,file=%s %s' % (
+                        linux.shellquote(secret_file), self._luks_image_opts(install_path))
+                    bash.bash_errorout(cmd)
+                    return True
+            out = bash.bash_errorout('/usr/bin/qemu-img info %s' % linux.shellquote(self._cbd_qemu_path(install_path)))
+            return 'file format: luks' in out
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)